src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/Microsoft.Extensions.DependencyInjection.Abstractions.csproj

```diff
@@ -10,9 +10,14 @@ Commonly Used Types:
 Microsoft.Extensions.DependencyInjection.IServiceCollection</PackageDescription>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Compile Include="$(CommonPath)Extensions\ParameterDefaultValue\ParameterDefaultValue.cs"
```

Comment:
Note that this file is included in both `DependencyInjection.Abstractions.csproj` and `DependencyInjection.csproj`. We probably need to do this to both projects.

Reply:
Applied to both projects